src/generate/EquipmentChoice_gen.hpp

```diff
@@ -0,0 +1,31 @@
+//
+// Created by Carolin on 17.05.2020.
+//
+
+#ifndef KICLIENT_EQUIPMENTCHOICE_GEN_HPP
+#define KICLIENT_EQUIPMENTCHOICE_GEN_HPP
+
+#include <vector>
+#include <util/UUID.hpp>
+#include <datatypes/gadgets/GadgetEnum.hpp>
+#include <datatypes/matchconfig/MatchConfig.hpp>
+#include <set>
+#include <datatypes/scenario/Scenario.hpp>
+#include <datatypes/character/CharacterInformation.hpp>
+
+class EquipmentChoice_gen {
```

Comment:
Macht es Sinn hier die übliche Naming Convention aufzubrechen?

Reply:
Habe das gemacht, damit es keine Überschneidungen in den File-Namen mit LibCommon gibt.